src/components/book.js

```diff
@@ -23,13 +27,22 @@ const Book = (prop) => {
 
   return (
 
-    <li key={item_id}>
-      {title}
-      {' | '}
-      {category}
-      <button type="button">Comments</button>
-      <button type="button" onClick={() => removeBookFromStore(item_id)}>Remove</button>
-      <button type="button">Edit</button>
+    <li key={item_id} className="Book">
+      <p className="book_title">{title}</p>
+      <p className="book_category">{category}</p>
+      <div id={item_id} className="book-options">
+        <a href="/#">Comments</a>
+        {' | '}
+        <a href="/#" onClick={() => removeBookFromStore(item_id)}>Remove</a>
+        {' | '}
+        <a href="/#">Edit</a>
+      </div>
+      <div>
+        <Percentage />
+      </div>
+      <div>
+        <Progress />
+      </div>
     </li>
```

Comment:
- [ ] The information about each book along with action buttons does not match the Zeplin design, kindly make sure to style them properly.
Your styling
![image](https://user-images.githubusercontent.com/33767951/140624623-e6ad8d73-71ef-42f3-93e9-bdc3f7b0d876.png)
Zeplin design
![image](https://user-images.githubusercontent.com/33767951/140624689-edc8b468-f916-4594-936f-52d1f664ec88.png)
- [ ] Vertical separators `|` are missing in each book, kindly add them.
Zeplin design
![image](https://user-images.githubusercontent.com/33767951/140620125-6f5d2ea7-9ec3-440b-9781-206766ce0989.png)
- [ ] Light-grey borders are missing from each book card, kindly add them.